### PR TITLE
#119 show more config details

### LINKED ISSUE #119: Difficult to review breach configuration
**Is your feature request related to a problem? Please describe.**

Currently when looking at the config, I can only see all the config numbers by navigating into each of the 4 settings. This seems a few more steps for what could be one.

Screenshot_2021-05-11-16-58-24-814

**Describe the solution you'd like**

2 Ideas, I'd probably lean towards the first:
1. Each configuration row has a subheader that just says `Temperature: x, duration y`. This would I would think is a simple enhancement so that I can see the config from one spot without a drastic rearrangement
2. Something closer to how I recall mobile's config for this. No deeper navigation, just all the fields exposed for each one on the one page.

**Describe alternatives you've considered**
n/a obvs more ideas are welcome!

**Additional context**

Coldchain docs: https://docs.msupply.foundation/en:cold_chain:settings
Mobile docs for comparison: https://docs.msupply.foundation/en:mobile:user_guide:vaccine

### PR DESCRIPTION
**Pull request by joshxg**
Fixes #119

### CODE REVIEW

File: src/common/constants/Dependency.ts [hunk 1]
```diff
@@ -1,5 +1,4 @@
 export enum DEPENDENCY {
-  MIGRATION = 'migrationService',
   LOCATOR = 'DependencyLocator',
```
Comment (mark-prins) [inline src/common/constants/Dependency.ts:2]: There's now only one usage of `DEPENDENCY.FORMAT_SERVICE` if the intention is to deprecate the enum, now might be a time to do it. I see that there's a lot of entries in the enum though, so unsure of the intention

**Review by mark-prins — APPROVED** [review]
that looks super good! Code looks fine too - nice to have some strings moved to translations, and I like the use of the dependencies and formatters.

It appears that the app version changes have snuck in here, so probably about time to merge the various branches in :)

File: src/common/constants/Dependency.ts (hunk 1 above)
Comment (joshxg) [reply src/common/constants/Dependency.ts:2]: I have been struggling a little to find the best way, but I think by using a discriminated union of the dependency keys, it is easier to type the return of the dep locator and also get autocomplete of the string, so yeah moving toward deprecating it

**Pull request merged**